looks like a dependency is broken... 

```
error: builder for '/nix/store/5na9pvl48ysqh3kcfcdsszv83w8vap9i-python3.10-pydyf-0.1.2.drv' failed with exit code 1;
       last 10 log lines:
       > >                   assert 0, (
       >                         f'Pixel ({x}, {y}) in {name}: '
       >                         f'reference rgba{reference}, got rgba{value}')
       > E                   AssertionError: Pixel (0, 5) in test_text: reference rgba(255, 255, 255), got rgba(223, 223, 223)
       >
       > tests/__init__.py:69: AssertionError
```

--- a/pkgs/development/python-modules/sentry-sdk/default.nix
+++ b/pkgs/development/python-modules/sentry-sdk/default.nix
@@ -37,6 +37,7 @@
 , trytond
 , urllib3
 , werkzeug
+, multidict
 }:
 
 buildPythonPackage rec {
@@ -88,6 +89,7 @@ buildPythonPackage rec {
     tornado
     trytond
     werkzeug
+    multidict
   ];
 
   doCheck = !stdenv.isDarwin;

--- a/pkgs/development/python-modules/weasyprint/default.nix
+++ b/pkgs/development/python-modules/weasyprint/default.nix
@@ -80,6 +80,8 @@ buildPythonPackage rec {
     # sensitive to sandbox environments
     "test_tab_size"
     "test_tabulation_character"
+    "test_linear_gradients_5"
+    "test_linear_gradients_12"
   ];
 
   FONTCONFIG_FILE = "${fontconfig.out}/etc/fonts/fonts.conf";

